lib/streamlit/elements/arrow_altair.py

```diff
@@ -319,11 +328,13 @@ def _generate_chart(
 
     chart = (
         getattr(
-            alt.Chart(data, width=width, height=height), "mark_" + chart_type.value
+            alt.Chart(data, width=width, height=height, usermeta=STREAMLIT_THEME),
```

Comment:
Hey @LukasMasuch!

I am a little bit confused trying to understand that part of the code, could you please explain this part 

If I understand correctly at the beginning of this file we register, but do not enable by default Streamlit theme, so it then could be easily enabled by Streamlit app developer with 

```python
with alt.themes.enable("streamlit"):
    st.arrow_altair_chart(...)
```

But on the other hand, on line 331 we use `usermeta=STREAMLIT_THEME` is it means that we always get attribute from "streamlit themed" chart?

Reply:
I will add a comment 👍  Short explanation: built-in charts will have it activated as default (`line_chart`, `bar_chart`, `area_chart`). That's why it is set here explicitly. Theoretically, the theme can also be activated for `altair_charts` via `alt.themes.enable` but this is not something we do not want to document for now. `alt.themes.enable` sets the theme globally, which can lead to unexpected problems between sessions.